strawberryfields/apps/qchem/dynamics.py

```diff
@@ -117,3 +122,92 @@ def op(t, Ul, w, q):
         sf.ops.Interferometer(Ul) | q
 
     return op
+
+
+def sample_fock(
+    input_state: list,
+    t: float,
+    Ul: np.ndarray,
+    w: np.ndarray,
+    n_samples: int = 1,
+    loss: float = 0.0,
+    cutoff: int = 15,
+) -> list:
+    r"""Generate samples for simulating vibrational quantum dynamics with a Fock input state.
+
+    **Example usage:**
+
+    >>> input_state = [0, 2]
+    >>> t = 10.0
+    >>> Ul = np.array([[0.707106781, -0.707106781],
+    >>>                [0.707106781, 0.707106781]])
+    >>> w = np.array([3914.92, 3787.59])
+    >>> n_samples = 5
+    >>> sample_fock(input_state, t, Ul, w, n_samples = n_samples)
```

Comment:
If we remove the default value of `n_samples`, then this should read 
` >>> sample_fock(input_state, t, Ul, w, n_samples )`